hack/build-antrea-linux-all.sh

```diff
@@ -161,8 +161,12 @@ cd -
 export NO_PULL=1
 if [ "$DISTRO" == "ubuntu" ]; then
     if $COVERAGE; then
+        make build-controller-ubuntu-coverage
+        make build-agent-ubuntu-coverage
         make build-ubuntu-coverage
```

Comment:
@jainpulkit22 @luolanzone Should we add a new mode to avoid building previous antrea-ubuntu image in this script? It will cost too much time to build all images in CI.

Reply:
@XinShuYang I am okay with it.

Reply:
@jainpulkit22 My question is, given that we have already built separate images for the controller and agent, why do we still need to build the original image? At least on the CI testbed, it is redundant for the deployment.

Reply:
It was decided that we have to build all the images for now, we should not remove the unified image, @XinShuYang so should i continue with the same or should i remove the unified image from here, or as you suggested we need different flag for test mode?

Reply:
I am okay with keeping a mode that can build all images. To avoid generating redundant images in CI, one way is to add a new mode that only generates separate images in this script. Also, the image delivery code in Jenkins scripts like test.sh and test-vmc.sh needs to be updated to make sure worker nodes and controller node use different image for testing.

Reply:
okay

Reply:
> I think for now we should keep generating the old image by default. I'm hoping that it doesn't increase build time too much, since base images are shared between agent image and unified image. We can have a flag to skip building the unified image.

Yes, with this flag we don't need to refactor the environment cleanup script in CI to remove redundant images.

Reply:
@jainpulkit22 Have you addressed this comment?